src/lightning/fabric/accelerators/tpu.py

```diff
@@ -49,14 +49,21 @@ def parse_devices(devices: Union[int, str, List[int]]) -> Union[int, List[int]]:
     @staticmethod
     def get_parallel_devices(devices: Union[int, List[int]]) -> List[torch.device]:
         """Gets parallel devices for the Accelerator."""
+        from torch_xla.experimental import pjrt
+
         devices = _parse_tpu_devices(devices)
-        # In XLA index 0 maps to CPU, in fact, a `xla_device()` with no arguments has index 1
-        # since the user passes a 0-based index, we need to adjust the indices
+        if pjrt.using_pjrt():
+            device_offset = 0
```

Comment:
The TPU devices in PJRT starts at idx 0

Reply:
To use CPU only, the env var `PJRT_DEVICE=CPU` is needed, and the `xm.xla_device()` would use CPU as an xla device.